package.json

```diff
@@ -19,7 +19,7 @@
     "@material-ui/core": "^4.12.3",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.58",
-    "@veupathdb/components": "^0.16.5",
+    "@veupathdb/components": "^0.16.6",
```

Comment:
Just noting that I upgraded @veupathdb/components to bring in https://github.com/VEuPathDB/web-components/pull/366